receiver/purefbreceiver/README.md

```diff
@@ -0,0 +1,51 @@
+# Pure Storage FlashBlade Receiver
+
+| Status                   |                     |
+| ------------------------ |---------------------|
+| Stability                | [in-development]    |
+| Supported pipeline types | metrics             |
+| Distributions            | [contrib]           |
+
+The Pure Storage FlashBlade receiver, receives metrics from Pure Storage internal services hosts.
+
+Supported pipeline types: metrics
+
+## Configuration
+
+The following settings are required:
+ -  `endpoint` (default: `http://172.31.60.207:9491/metrics/array`): The URL of the scraper selected endpoint
+
+Example:
+
+```yaml
+extensions:
+  bearertokenauth/fb01:
+    token: "..."
+
+receivers:
+  purefb:
+    endpoint: http://172.31.60.207:9491/metrics
+
+  purefb/with_custom_intervals:
+    endpoint: http://172.31.60.207:9491/metrics
+    arrays:
```

Comment:
Should this list all endpoints? The doc should also mention that only endpoints explicitly added here will be scraped.

Reply:
+1